src/dune_rules/pkg_toolchain.mli

```diff
@@ -4,6 +4,8 @@ open Import
     versions. Creates the directory if it doesn't already exist. *)
 val base_dir : unit -> Path.Outside_build_dir.t
 
+val compiler_package_names : Dune_lang.Package_name.t list
```

Comment:
A documentation comment would be nice here. But tbh, I am not sure this is the right place for this to live. Maybe it can be in `src/dune_pkg` from which both the toolchains as well as the dev-tools can use it?